tests/integration/worker_tests.rs

```diff
@@ -125,3 +126,9 @@ itest!(node_worker_auto_exits {
   output: "workers/node_worker_auto_exits.mjs.out",
   exit_code: 0,
 });
+
+itest_flaky!(node_worker_message_port {
+  args: "run --quiet --allow-read workers/node_worker_message_port.mjs",
+  output: "workers/node_worker_message_port.mjs.out",
+  exit_code: 0,
+});
```

Comment:
So it passes sometimes?

Reply:
I marked it as flaky because the order is not predictable. But it doesn't match the output of Node yet.